Tests/unit/core/test_message_dispatcher.cpp

```diff
@@ -251,7 +251,7 @@ namespace Tests {
     }
 
     //doorbell (socket) is not quite working inside test suite
-    TEST_F(Core_MessageDispatcher, DISABLED_ReaderShouldWaitUntillRingBells)
+    TEST_F(Core_MessageDispatcher, ReaderShouldWaitUntillRingBells)
```

Comment:
It seems you have disable test_message_dispatcher.cpp in the cmakelist. Is this due to enabling these tests ?

Reply:
Good catch, but no, it was because test_message_dispatcher wasn't building at all:
```
/home/runner/work/Thunder/Thunder/Thunder/Tests/unit/core/test_message_dispatcher.cpp:71:31: error: ‘MessageDispatcherType’ is not a member of ‘WPEFramework::Core’
   71 |         std::unique_ptr<Core::MessageDispatcherType<METADATA_SIZE, DATA_SIZE>> _dispatcher;
      |                               ^~~~~~~~~~~~~~~~~~~~~
```
No need to enable this test tho, you're right, will keep it disabled.